tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
I think it's certainly harmless for RC1 so feel free to ignore this for now, but @lodejard @davidfowl doesn't this look wrong to you? The strong-named-ness of a build-time dependency shouldn't affect the main project should it?